doc/source/templates/01_batch_inference/batch_inference.ipynb

```diff
@@ -0,0 +1,331 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "6fbc3e3c",
+   "metadata": {},
+   "source": [
+    "# Scaling Batch Inference with Ray Data\n",
+    "\n",
+    "This template is a quickstart to using [Ray Data](https://docs.ray.io/en/latest/data/dataset.html) for batch inference. Ray Data is one of many libraries under the [Ray AI Runtime](https://docs.ray.io/en/latest/ray-air/getting-started.html). See [this blog post](https://www.anyscale.com/blog/model-batch-inference-in-ray-actors-actorpool-and-datasets) for more information on why and how you should perform batch inference with Ray!\n",
+    "\n",
+    "This template walks through GPU batch prediction on an image dataset using a PyTorch model, but the framework and data format are there just to help you build your own application!\n",
+    "\n",
+    "At a high level, this template will:\n",
+    "1. [Load your dataset using Ray Data.](https://docs.ray.io/en/latest/data/creating-datasets.html)\n",
+    "2. [Preprocess your dataset before feeding it to your model.](https://docs.ray.io/en/latest/data/transforming-datasets.html)\n",
+    "3. [Initialize your model and perform inference on a shard of your dataset with a remote actor.](https://docs.ray.io/en/latest/data/transforming-datasets.html#callable-class-udfs)\n",
+    "4. [Save your prediction results.](https://docs.ray.io/en/latest/data/api/input_output.html)\n",
+    "\n",
+    "> Slot in your code below wherever you see the ✂️ icon to build a many model training Ray application off of this template!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "065e7765",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "import numpy as np\n",
+    "import tempfile\n",
+    "from typing import Dict\n",
+    "\n",
+    "import ray\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c99f142a",
+   "metadata": {},
+   "source": [
+    ">✂️ Replace these values depending on the template size you picked!\n",
+    ">\n",
+    ">For example, for the larger scale template with 4 GPU nodes, you may want to use 4 workers, each using 1 GPU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9aa792fc",
+   "metadata": {
+    "tags": [
+     "small"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "NUM_WORKERS: int = 1\n",
+    "\n",
+    "USE_GPU: bool = True\n",
+    "NUM_GPUS_PER_WORKER: float = 1\n"
```

Comment:
This cell is repeated twice.

Reply:
This is how I'm getting around loading a small vs. large version of the template. On product side, we can run some post init command to process the notebook:
- Filter out all notebook cells tagged as `large`. Vice versa for large scale examples. This way, the configurations get set to use the small example defaults.

```bash
jupyter nbconvert --TagRemovePreprocessor.remove_input_tags='large'
      --to notebook --output batch_inference.ipynb batch_inference.ipynb
```

How does that sound @ericl?

Reply:
Yes, sounds good!